src/Raven.Server/Documents/Sharding/Handlers/ReshardingHandler.cs

```diff
@@ -1,22 +1,28 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Backups;
 using Raven.Client.Exceptions.Database;
+using Raven.Server.Documents.Operations;
+using Raven.Server.Json;
 using Raven.Server.Routing;
 using Raven.Server.ServerWide.Context;
 using Raven.Server.Utils;
 using Raven.Server.Web;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server.Collections;
 
 namespace Raven.Server.Documents.Sharding.Handlers
 {
     public class ReshardingHandler : ServerRequestHandler
     {
-        [RavenAction("/admin/resharding/start", "POST", AuthorizationStatus.Operator)]
+        [RavenAction("/admin/resharding/start", "GET", AuthorizationStatus.Operator)]
         public async Task StartResharding()
         {
             var database = GetStringQueryString("database");
-            var bucket = GetIntValueQueryString("bucket").Value;
-            var from = GetIntValueQueryString("from", required: false);
-            var to = GetIntValueQueryString("to").Value;
+            var buckets = GetStringValuesQueryString("bucket");
```

Comment:
Is it new way of passing ranges? I think this is first time I see it.

Reply:
I wanted to support nicely both single bucket and a range... but tell me how you want that to be and we can arrange it :)

Reply:
I'd say specify range fromBucket toBucket + we allow to supply from A to A.

Reply:
https://github.com/ravendb/ravendb/blob/c906b43b2cde6346c0704e4f4a93f632519c16a2/src/Raven.Client/Documents/Session/SessionTimeSeriesBase.cs#L99

Reply:
okay